Formula/binutils.rb

```diff
@@ -20,25 +20,37 @@ class Binutils < Formula
 
   uses_from_macos "zlib"
 
+  on_linux do
+    depends_on "glibc@2.13" => :build
+  end
+
   def install
-    system "./configure", "--disable-debug",
-                          "--disable-dependency-tracking",
-                          "--enable-deterministic-archives",
-                          "--prefix=#{prefix}",
-                          "--infodir=#{info}",
-                          "--mandir=#{man}",
-                          "--disable-werror",
-                          "--enable-interwork",
-                          "--enable-multilib",
-                          "--enable-64-bit-bfd",
-                          "--enable-gold",
-                          "--enable-plugins",
-                          "--enable-targets=all",
-                          "--with-system-zlib",
-                          "--disable-nls"
-    system "make"
-    system "make", "install"
-    bin.install_symlink "ld.gold" => "gold"
+    # Fix: error: 'LONG_MIN' undeclared
+    ENV.append "CFLAGS", "-DHAVE_LIMITS_H -DHAVE_FCNTL_H" unless OS.mac?
+
+    args = [
+      "--disable-debug",
+      "--disable-dependency-tracking",
+      "--enable-deterministic-archives",
+      "--prefix=#{prefix}",
+      "--infodir=#{info}",
+      "--mandir=#{man}",
+      "--disable-werror",
+      "--enable-interwork",
+      "--enable-multilib",
+      "--enable-64-bit-bfd",
+      "--enable-plugins",
+      "--enable-targets=all",
+      "--with-system-zlib",
+      "--disable-nls",
+      "--disable-gold",
+    ]
+    system "./configure", *args
+    # Pass MAKEINFO=true to disable generation of HTML documentation.
+    # This avoids a build time dependency on texinfo.
+    system "make", "MAKEINFO=true"
+    system "make", "install", "MAKEINFO=true"
```

Comment:
Do we want to set `MAKEINFO=true` on macOS as well? macOS ships a `/usr/bin/makeinfo`.

Reply:
You’re right, this should be Linux-only, I’ll change that.  I suppose it doesn’t really matter for `gold` though because that is Linux-only.